BookingsApi/BookingsApi.DAL/Queries/GetAllocationHearingsBySearchQuery.cs

```diff
@@ -42,11 +47,11 @@ public class GetAllocationHearingsBySearchQueryHandler : IQueryHandler<GetAlloca
     {
         private readonly BookingsDbContext _context;
         private readonly bool _isTest;
-
-        public GetAllocationHearingsBySearchQueryHandler(BookingsDbContext context, bool isTest = false)
+        
+        public GetAllocationHearingsBySearchQueryHandler(BookingsDbContext context, IOptions<SettingsConfiguration> options)
         {
             _context = context;
-            _isTest = isTest;
+            _isTest = options.Value.IsTest;
```

Comment:
Is there any other way this can be handled for tests - may be create test data as setup and clean up in the tear down?

Reply:
I'm not sure how to tackle this one, the query was written to ignore generic hearings if the flag is on. Our AC tests create hearings with the case type generic and so would not be returned.

Ideally this is the kind of API test that I would have in a non-AC test project because toggling this setting before and after tests doesn't feel right